Examples/Catalog/macOS/Source/ViewControllers/TopicCollectionViewController.swift

```diff
@@ -0,0 +1,31 @@
+import Cocoa
+import CatalogCore
+import Pilot
+import PilotUI
+
+public final class TopicCollectionViewController: CollectionViewController {
+    
+    // MARK: Public
+    
+    public static func make(with context: CatalogContext) -> TopicCollectionViewController {
```

Comment:
random idea, but since we do a lot of things like this in our app it might be nice to provide an optional protocol CVCs can conform to with some static funcs which if conformed to provide a convenience init(context:) initializer

```
static func defaultModel() -> ModelCollection
static func defaultLayout() -> NSCollectionViewLayout
static func viewBinder() -> ViewBindingProvider
```

Reply:
nice idea! filed https://github.com/dropbox/pilot/issues/110